test-js/integration.js

```diff
@@ -58,17 +59,26 @@ describe("integration tests", function() {
   }));
 
   it("refuses to init without --force", run(function*(sh, pulp, assert, temp) {
-    touch.sync(path.join(temp, "bower.json"));
-    const [_, err] = yield pulp("init", null, { expectedExitCode: 1 });
-    assert.equal(err.trim(), initWithoutForce);
+    for (var idx in filesToOverwrite) {
+      let file = path.join(temp, filesToOverwrite[idx])
+        , dir  = path.dirname(file)
+      if (!fs.existsSync(dir)) { yield sh(`mkdir ${dir}`); }
```

Comment:
Same here, could this be `fs.mkdirSync`? (I realise I might not have used `fs` absolutely everywhere previously but I think it's probably what we should be doing)